tests/lib/json/src/main.c

```diff
@@ -22,6 +22,10 @@ struct test_struct {
 	struct test_nested some_nested_struct;
 	int some_array[16];
 	size_t some_array_len;
+	bool another_bxxl;		 /* JSON field: "another_b!@l" */
```

Comment:
Might be a good idea to have another field here, whose JSON key is a reserved keyword in C. (In OCF, we had an "if" key for interface when it used to use JSON encoding, as an example.)